src/resource.jl

```diff
@@ -134,38 +184,47 @@ const FETCH_LOCKS = [ReentrantLock() for _ = 1:fetch_locks]
 const FETCH_FAILS = [Set{String}() for _ = 1:fetch_locks]
 const FETCH_DICTS = [Dict{String,Event}() for _ = 1:fetch_locks]
 
-function fetch(resource::String; servers=STORAGE_SERVERS)
-    path = "cache" * resource
-    isfile(path) && return path
-    isempty(servers) && throw(@error "fetch called with no servers" resource=resource)
-    # make sure only one thread fetches path
-    i = (hash(path, FETCH_SEED) % fetch_locks) + 1
+function fetch(resource::String; servers=config.storage_servers)
+    if hit!(config.cache, resource[2:end])
```

Comment:
```suggestion
    if hit!(config.cache, lstrip(resource, '/'))
```
There should probably only be a single leading slash, but let's make sure we don't accidentally give access to things at the root of the file system.